stdlib/private/StdlibCollectionUnittest/LoggingWrappers.swift.gyb

```diff
@@ -139,16 +139,16 @@ public struct LoggingRangeReplaceableCollection<
     return base[position]
   }
 
-  public mutating func replaceSubrange<
+  public mutating func replace<
     C : Collection where C.Iterator.Element == Base.Iterator.Element
   >(
-    bounds: Range<Base.Index>, with newElements: C
+    subRange bounds: Range<Base.Index>, with newElements: C
```

Comment:
Please don't change the standard library API.  It was already audited and it conforms to the guidelines.